content/docs/user-guide/setup-dropbox-remote.md

```diff
@@ -0,0 +1,82 @@
+# Setup a Dropbox DVC Remote
+
+In this guide we explain the existing ways to setup Dropbox
+[remote storage](/doc/command-reference/remote) for your <abbr>DVC
+projects</abbr>, along with the different benefits each one brings.
+
+## Quick start
+
+To start using a Google Drive remote, you only need to add it with a
+[valid URL format](#url-format). Then use any DVC command that needs to connect
+to it (e.g. `dvc pull` or `dvc push` once there's tracked data to synchronize).
+For example:
+
+```dvc
+$ dvc add data
+...
+$ dvc remote add --default myremote \
+                           gdrive://0AIac4JZqHhKmUk9PDA/dvcstore
+$ dvc push
+
+Go to the following link in your browser:
+
+    https://www.dropbox.com/oauth2/authorize?response_type=code&client_id=<client-id>&token_access_type=offline # ... copy this link
+
+Enter verification code: # <- enter resulting code
+```
+
+See [Authorization](#authorization) for more details.
+
+## URL format
+
+You can indicate any directory name that will be created under the
+`DROPBOX_HOME/Applications/dvc`, e.g.
+`DROPBOX_HOME/Applications/dvc/my-project`. Make sure this name is not already
```

Comment:
it can be used by another project technically, some people use the same remote for all projects

Reply:
True, it could be.